mmv1/products/compute/api.yaml

```diff
@@ -7528,7 +7530,7 @@ objects:
         description: |
           TargetPool resources to which instances in the instanceGroup field are
           added. The target pools automatically apply to all of the instances in
-          the managed instance group.
+          the managed instance group.'
```

Comment:
Why is this added?

Reply:
Also, technically no changes to this api.yaml are needed at all. we could consider removing all of the changes to this file just for simplicity's sake.